src/Components/TruckCard/TruckCard.js

```diff
@@ -16,30 +16,6 @@ const TruckCard = (props) => {
   let [upVote, setupVote] = useState(parseInt(props.truck.up_rating))
   const [isUpvoted, setIsUpvoted] = useState(false)
   
```

Comment:
I think taking out the distance functionality is really smart for right now, and we're all behind this! We could certainly get it working given more time, but as it stands we're losing valuable time continuing to focus on this. We can always return to it at the end of the project if it feels like we have room to play with.